libcxx/include/stack

```diff
@@ -276,10 +276,18 @@ public:
   _LIBCPP_NODISCARD _LIBCPP_HIDE_FROM_ABI const _Container& __get_container() const { return c; }
```

Comment:
```suggestion
```

This is probably no longer necessary (above also). I didn't verify that claim it though.

Reply:
It's used in `std::format` https://github.com/llvm/llvm-project/blob/main/libcxx/include/__format/container_adaptor.h#L53